transformations/mappings/index.md

```diff
@@ -133,7 +137,34 @@ Clone table:
 ![Table Clone](/transformations/mappings/table-clone.png)
 
 The `CLONE` mapping will execute almost instantly for a table of any size (typically under 10 seconds) 
-as it physically does not move any data. 
+as it physically does not move any data.
+
+On the other hand, you can use [Read-Only Input Mapping](/transformations/mappings/#read-only-input-mapping) which makes available all buckets and tables with read access,
+so there is no need to clone the tables into a new schema, you can simply read from these buckets and tables in the trace.
+This function is automatically enabled in the transformations.
+
+##### Read-Only Input Mapping
+For using **Read-Only Input Mapping** there is no need to set anything else and for transformations, tables are automatically accessible in transformation.
+This also applies to linked buckets. Here you have to pay attention to the fact that buckets and tables belong
+to another project, so you have to access e.g. the database of another project in dependence on the backend.
+
+{: .image-popup}
+![Storage](/transformations/mappings/storage.png)
+
+The way how it works is that you have read access to all tables that exist in the storage in your project.
```

Comment:
```suggestion
You have read access to all tables that exist in your project's Storage directly on the underlying backend. However, this means you need to use the internal ID of bucket (`in.c-main` instead of `main` as you see in the UI). 
```